docs/networks/mainnet/parameters.md

```diff
@@ -46,20 +46,20 @@ This gateway is intended for development purposes. We do not guarantee any SLA,
 
 :::
 
-## Add LUKSO Mainnet (MetaMask...)
+## Add LUKSO Mainnet to Wallets
 
-<AddNetworkButton networkName="mainnet"/>
-
-You can add LUKSO as a custom network in your existing wallet (MetaMask, Rainbow, Coinbase Wallet, Trust Wallet, etc.) with the parameters below:
+You can add the LUKSO Mainnet to any of your existing wallets like [MetaMask](https://metamask.io/), [Rabby](https://rabby.io/), [Rainbow](https://rainbow.me/), [Coinbase Wallet](https://www.coinbase.com/de/wallet), [Trust Wallet](https://trustwallet.com/de), and others using the following parameters or button:
 
 | Setting               | Value                                             |
 | --------------------- | ------------------------------------------------- |
-| Network Name          | LUKSO                                             |
```

Comment:
Let's stick to LUKSO only please.

98% of users will only use mainnet so it is not relevant for them to see "mainnet" everytime they use their extension.

We left to "LUKSO" only on purpose.

(Please check the MetaMask add button PR too I might have missed that too if you added "mainnet" to the work)

Reply:
Alright, that makes sense.